app/scripts/modules/amazon/serverGroup/details/serverGroupDetails.html

```diff
@@ -89,6 +89,11 @@ <h3 select-on-dbl-click>
         <render-if-feature feature="entityTags">
           <entity-source metadata="ctrl.serverGroup.entityTags.creationMetadata"></entity-source>
         </render-if-feature>
+        <dt ng-if="ctrl.state.changesAvailable">Changes</dt>
+        <dd ng-if="ctrl.state.changesAvailable">
+          <view-changes-link commits="ctrl.commits" jar-diffs="ctrl.jarDiffs" link-text="view changes"
+                             name-item="ctrl.serverGroup"></view-changes-link>
+        </dd>
```

Comment:
probably want to remove the `<dt>` and `<dd>` and move the `<view-changes-link>` into the `<render-if-feature>` body above